scripts/docsAndLastDateEdited.mjs

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env node
+
+import { glob } from 'glob10';
+import { statSync, writeFile } from 'fs';
+import { stringify } from 'csv-stringify';
+
+const allDocs = await glob('src/content/docs/**/*.mdx', {
+  ignore: '**/index.mdx',
+});
+
+const allDocsAndDates = allDocs.map((doc) => {
+  // mtime is the last modified time of a particular file
+  // https://nodejs.org/api/fs.html#statsmtime
+  const modifiedDate = statSync(doc).mtime;
```

Comment:
i think we need to use git data and not the file system. using the file system will only tell you last time it was modified on machine's file system and not edit history via git commits